### PR TITLE
✨ FEAT: Use yarn

### PR DESCRIPTION
**Pull request by kenjinp**
it was difficult to upgrade my packages for another task, so I though it's time to keep this repo in line with https://github.com/BlueBrain/nexus-js and convert to yarn. 

- fixes security issues
- upgrades packages
- fixes resulting type errors
- updates snapshots

### CODE REVIEW

File: src/shared/components/FileUpload/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Storage, Project, NexusFile } from '@bbp/nexus-sdk-legacy';
 import { StorageCommon } from '@bbp/nexus-sdk-legacy/lib/Storage/types';
 import { CreateFileOptions } from '@bbp/nexus-sdk-legacy/lib/File/types';
 import { labelOf } from '../../utils';
-import { UploadFile } from 'antd/lib/upload/interface';
+import { UploadFile, RcCustomRequestOptions } from 'antd/lib/upload/interface';
```
Comment (julienmachon) [inline src/shared/components/FileUpload/index.tsx]: What does this have to do with the yarn migration?

File: src/shared/components/FileUpload/index.tsx [hunk 2]
```diff
@@ -74,15 +74,19 @@ const FileUploader: React.FunctionComponent<FileUploaderProps> = ({
...
-  const handleFileUpload = async (customFileRequest: CustomFileRequest) => {
-    try {
-      const options = storageId ? { storage: storageId } : undefined;
-      const nexusFile = await onFileUpload(customFileRequest.file, options);
-      setRecentlyUploadFileList([...recentlyUploadedFileList, nexusFile]);
-      customFileRequest.onSuccess('Successfully uploaded file');
-    } catch (error) {
-      customFileRequest.onError(error);
-    }
+  const handleFileUpload = (customFileRequest: RcCustomRequestOptions) => {
```
Comment (julienmachon) [inline src/shared/components/FileUpload/index.tsx]: Same comment for this one

File: src/shared/components/FileUpload/index.tsx (hunk 1 above)
Comment (kenjinp) [reply src/shared/components/FileUpload/index.tsx]: The types changed, probably because of version differences

File: src/shared/components/FileUpload/index.tsx (hunk 2 above)
Comment (kenjinp) [reply src/shared/components/FileUpload/index.tsx]: version differences, I ran `yarn audit fix` its probably why

File: src/shared/components/FileUpload/index.tsx (hunk 1 above)
Comment (julienmachon) [reply src/shared/components/FileUpload/index.tsx]: gotcha!
